ecommerce/colaraz_features/api/serializers.py

```diff
@@ -0,0 +1,64 @@
+"""
+Serializers for colaraz API.
+"""
+from django.conf import settings
+
+from oscar.core.loading import get_model
+from rest_framework import serializers
+
+from ecommerce.colaraz_features.api.helpers import (
+    create_site_configurations,
+    create_site_theme,
+    get_or_create_site,
+    get_partner,
+)
+
+Partner = get_model('partner', 'Partner')
+DEFAULT_FROM_EMAIL = getattr(settings, 'OSCAR_FROM_EMAIL', '')
+DEFAULT_SUPPORT_EMAIL = getattr(settings, 'DEFAULT_PAYMENT_SUPPORT_EMAIL', '')
+
+class SiteOrgSerializer(serializers.Serializer):
+    lms_site_url = serializers.URLField(required=True)
```

Comment:
```suggestion
    lms_url_root = serializers.URLField(required=True)
```

Reply:
`lms_url_root` is being used in for base site e.g: `courses.colaraz.com`, while the logic behind `lms_site_url` related to multisite e.g: `polymath.courses.colaraz.com and insigniabizedu.courses.colaraz.com`